src/components/user/event.tsx

```diff
@@ -101,7 +101,8 @@ export function Event({
           thread: renderThread(event.thread),
           comment: (
             <Link href={`/${event.comment.id}`} forceNoCSR>
-              {`${strings.entities.comment} (${event.comment.id})`}
+              {strings.entities.comment}
```

Comment:
I'd give it a space in between the title and the id `{' '}`.

Reply:
(same for the other `sub`s)

Reply:
I agree, that looks better!